java/ant.hints/src/org/netbeans/modules/ant/hints/errors/EnablePreviewAntProj.java

```diff
@@ -195,7 +166,7 @@ public EditableProperties run() throws IOException {
         }
     }
 
-    private boolean isAntProject(Project prj) {
+    private static boolean isAntProject(Project prj) {
         if (prj == null) {
```

Comment:
this is a privete method? can't you guarantee thot no callsite passes null? then you could get rid of this if statment

Reply:
Well, `FileOwnerQuery.getOwner` can return `null`. So, there must be a null check somewhere. I don't really see an advantage in doing that before calling the method? Is there some? (Frankly, even if the method was public, I'd probably make it accept nulls, to make its use as convenient as possible, so that callers would not have to additionally check for `null`. I am not sure what's the issue with a method checking for `null`.)